contents/blog/what-to-ask-in-interviews.md

```diff
@@ -0,0 +1,114 @@
+
+"Are you [default alive](http://www.paulgraham.com/aord.html)?"
+
+This means that the company whilst not profitable, is growing fast enough that it will become profitable ahead of running out of money.
+
+Avoid:
+
+* Companies that don't know this or don't care about it, given [the current climate](https://techcrunch.com/2022/05/06/the-venture-slowdown-isnt-coming-its-here/) - this may change in an easier fundraising environment for early stage, capital-intense business models
+* Companies that you don't believe could get to default alive due to a lack of time and speed to get stuff done, if they aren't
+
+There are exceptions. For example, very low revenue numbers may lead to unrealistic growth rates - if I went from $1 to $2 of revenue yesterday, we'd be pretty darn rich if we kept that growth rate up.
+
+"What's your runway? What assumptions are you making to calculate this?"
+
+If a company doesn't have product-market fit, spending should be minimal so there's as much time as possible to find it. Spending a lot and having a big team just makes it harder to change direction rapidly.
+
+If a company has strong product market fit _and_ has a ton of traction, a shorter runway is ok.
+
+Avoid: 
+
+* Companies that assume wild increases in revenue to survive, within a short timeframe
+
+## What's the culture like?
+
+"What are the company values, and why? Can you give me some specific ways you've followed them?"
+"Who decides what to build?"
+"What do you look for when you hire?"
+
+We get the above a bit more often, they're good questions.
+
+## How strong is the team?
+
+"Can you tell me about the team I'd be working with most closely?"
+"How do you compensate the team?"
+"How do you attract and retain really strong hires?"
+
+You are trying to understand if they view engineering as a cost center (aka higher propensity you'll be miserable there!) or not.
+
+Beyond that, if you want to learn in this role - do you believe this company is able to attract people who are as good as or better than you?
+
+## What's in store in the future?
+
+"What motivates you?"
+"What are you most proud of so far?"
+"Do you plan to sell the company?"
+"Do you believe in secondaries? Would you offer these to your team, if possible?"
+
+You can't guarantee that plans won't change, but this gives you some indication of what is in store and it helps you understand the founders' motivation better.
+
+"What's the company strategy and why?"
+
+You are looking for a simple answer - otherwise the company and the product will end up a bit of a mess.
+
+"How do you and the other founders avoid falling out?"
+
+One of the most common ways for an early stage company to die is that the founders break up. Hopefully they've put some proactive thought into their relationship.
+
+"What keeps you up at night?"
+
+This will reveal perhaps what the hardest challenge will be, and where you could add the most value if you joined 
+
+## Do I want to work here?
```

Comment:
As above, I'd take this out and make this a more startup-focused piece rather than engineer-focused, as it reads much more as the former than the latter.

Reply:
changed the cocnlusion to incorporate this feedback